src/constants/assetsConstants.js

```diff
@@ -91,12 +96,27 @@ export const DEFAULT_ACCOUNTS_ASSETS_DATA_KEY = 'default';
 export const VISIBLE_NUMBER_DECIMALS = 18;
 
 export const POPULAR_EXCHANGE_TOKENS = [
-  ETH, PLR, LINK, BAT, PPT, AE, DAI, KNC, MKR, USDT, VERI, BNT, OMG, WAX, ZIL, ZRX,
+  ETH,
+  PLR,
+  LINK,
+  BAT,
+  PPT,
+  AE,
+  DAI,
+  KNC,
+  MKR,
+  USDT,
+  VERI,
+  BNT,
+  OMG,
+  WAX,
+  ZIL,
+  ZRX,
 ];
 
-export const HIGH_VALUE_TOKENS = [
-  YFI, WBTC, sBTC, BTC, iBTC, renBTC, tBTC, OBTC, wBTC, testBTC,
-];
+export const HIGH_VALUE_TOKENS = [YFI, WBTC, sBTC, BTC, iBTC, renBTC, tBTC, OBTC, wBTC, testBTC];
+
+export const RAMP_CURRENCY_TOKENS = [ETH, DAI, PLR, USDC, USDT, MATIC, MATIC_DAI, MATIC_USDC, XDAI];
```

Comment:
Consider moving to `configs/rampConfig` (file does not exist yet).

Reply:
I guess it does not makes sense to create new file for currency.Since we already have file where we are maintaining assets. Let's use this only. What say ?

Reply:
For consistency with other service integrations like `rari` & `aave` I would recomment not mixing service integration specific config with general asset constants.